plan_runner/manipulation_station_simulator.py

```diff
@@ -59,12 +54,15 @@ def RunSimulation(self, plans_list, gripper_setpoint_list,
                       extra_time=0, real_time_rate=1.0, q0_kuka=np.zeros(7), is_visualizing=True):
         '''
         Constructs a Diagram that sends commands to ManipulationStation.
-        :param plans_list:
-        :param gripper_setpoint_list:
-        :param extra_time:
-        :param real_time_rate:
-        :param q0_kuka:
-        :return:
+        :param plans_list: A list of Plans to be executed.
+        :param gripper_setpoint_list: A list of gripper setpoints. Each setpoint corresponds to a Plan.
+        :param extra_time: Time in seconds that the simulation is run,
+                            in addition to the sum of the durations of all plans.
+        :param real_time_rate: 1.0 means realtime; 0 means as fast as possible.
+        :param q0_kuka: initial configuration of the robot.
+        :param is_visualizing: if true, adds MeshcatVisualizer to the Diagram. It should be set to False
+                                when running tests.
+        :return: logs of robot configuration and MultibodyPlant, generated by simulation.
         '''
```

Comment:
Use double quotes for docstrings
```suggestion
        """
```